pkg/platform/authentication/auth.go

```diff
@@ -49,6 +50,8 @@ type Configurable interface {
 	Close() error
 }
 
+const deviceCodeConfigKey = "deviceCode"
```

Comment:
I think we wanted to standardize our config keys so this should be `device.code` @Naatan can you confirm?

Reply:
Good point! But in this case this shouldn't be stored in the config at all.